packages/docusaurus-plugin-content-blog/src/frontMatter.ts

```diff
@@ -4,14 +4,15 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  */
-
 import {
+  ContentVisibilitySchema,
+  FrontMatterAuthorErrorMessage,
```

Comment:
FrontMatterAuthorErrorMessage is specific to the blog plugin and not shared, so it should rather stay there

CF the error message, it's clearly not intended to be used by the docs plugin:


```js
const FrontMatterAuthorErrorMessage =
  '{{#label}} does not look like a valid blog post author. Please use an author key or an author object (with a key and/or name).';
```